covid_graphs/covid_graphs/pb/atg_prediction.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+message Date {
+  uint32 day = 1;
+  uint32 month = 2;
+  uint32 year = 3;
+}
+
+message AtgTraceGenerator {
```

Comment:
I feel quite strongly that this should not be called a "generator", and especially because this is an interface we should choose a good name. I suggest something along `AtgParameters` or `AtgModelParameters` (I'm fine with `...Params` as well).